packages/avatar-creator/src/components/MmlButtons.tsx

```diff
@@ -17,14 +17,18 @@ export default function MmlButtons({
   onSave,
   isLoading,
 }: {
-  setOverlayActive: (value: boolean) => void;
+  setOverlayActive: (value: string) => void;
```

Comment:
Can you make this an `enum` please? Just so that it's clear from both sides of the prop what the expected values are.